cmd/neofs-adm/README.md

```diff
@@ -5,14 +5,14 @@
 Admin tool provides an easier way to deploy and maintain private installation
 of NeoFS. Private installation provides a set of N3 consensus nodes, NeoFS 
 Alphabet, and Storage nodes. Admin tool generates consensus keys, initializes 
-side chain, and provides functions to update the network and register new
+the sidechain, and provides functions to update the network and register new
```

Comment:
could you, please, provide the rule describing the difference b/w "side chain", "the sidechain" and "side chain"? see all in that PR

Reply:
I've discussed it with @realloc and we've decided to write `sidechain`, `mainchain` etc in one word when it is one entity (well, in our context everywhere). Definite article is used with an entity that is the only one in a certain context, so I'd recommend to use `the` with `sidechain`. Unless `sidechain` is an attribute to some noun, eg. `sidechain contract` -- here we use article as needed for the noun (a ... contract, the .. contract, ... contracts)